### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by SamanthaPam**
**Changes:**
- Logo and Home page image updated.
- Footer and header (with the name and logo) updated.
- Home page design and content.
- css style added.

**Jira instance:**
https://comp299-group3.atlassian.net/browse/CG-3?atlOrigin=eyJpIjoiYTdiZDU0NjU4NjNmNDY4MGFjZjk5YjJjMjM2ZGE1OTIiLCJwIjoiaiJ9

### CODE REVIEW

File: client/images/home image.png
Comment (MaxwelSantana) [inline client/images/home image.png:1]: - Please put the images inside an Assets folder. Should be "Assets/images" (Assets with capital letter).
- Also avoid spaces on the image files. I suggest rename "home image.png" to "home-image.png"

File: server/views/content/index.ejs [hunk 1]
```diff
@@ -1,15 +1,29 @@
 <% include ../partials/header.ejs %>
 
     <!-- MAIN CONTENT -->
-    <div class="container">
+    <div style="background-color:burlywood;min-height: 400px;width: 100%;">
```
Comment (MaxwelSantana) [inline server/views/content/index.ejs]: Why remove container class? I think this is why the vertical scroll is showing

**Review by MaxwelSantana — CHANGES_REQUESTED** [review]
Some comments only. Overall LGTM! Great Job!

File: client/images/home image.png
Comment (SamanthaPam) [reply client/images/home image.png:1]: Agree with you Max. Changes commited

File: server/views/content/index.ejs (hunk 1 above)
Comment (SamanthaPam) [reply server/views/content/index.ejs]: I deleted because It didn't cover 100% of the page but now I fixed it.

**Review by MaxwelSantana — APPROVED** [review]
LGTM!

**Pull request merged**